### DISCUSSION TITLE
Why is there no sync gauge ?

### DISCUSSION THREAD

**Discussion by eddycharly** [open-telemetry/opentelemetry-go#3517] [General]
Might be a dumb question but I wonder why Gauges are only async ?

**Comment by MrAlias** [comment]
This project is an implementation of the OpenTelemetry specification, it would need to be specified there before we would add it here.

Please take a look at https://github.com/open-telemetry/opentelemetry-specification/issues/2318, where the discussion of adding a synchronous gauge to the specification is happening.

**Reply by eddycharly** [reply]
Thanks !

**Reply by eddycharly** [reply]
Another question, what is the difference between an async gauge and and an async up/down counter ?
